Update engineeringColleges.js

pankaj892 opened the pull request:
Added new college name and formatted codebase

## Description
Added new colleges name and removed errors due to single quotes by escaping characters and also formatted the codebase to be concise and readable

---
## Issue Ticket Number
Fixes #(issue_number)

---

# Checklist:
- [x] I have followed the contributing guidelines of this project as mentioned in [CONTRIBUTING.md](/CONTRIBUTING.md)
- [x] I have cross-checked that the dataset I'm uploading is not present in the project
- [x] I have cross-checked that the dataset is present in the correct folder

nikhil25803 commented:
Hey @pankaj892 

Can't accept this PR because according to the PR/MR policy of Hactoberfest, repos that encourage simplistic PR/MRS (like adding a name or profile to a list or arbitrarily curating content) will be excluded from Hacktoberfest.

Hope to see some quality changes from your side next time. Remember: quantity is fun, quality is key.

pankaj892 commented:
@nikhil25803 i removed duplicate names and also formatted the code and also added some college names I thought that was a lot for a PR nevermind will try to work on something else in the repo
